### ISSUE TITLE
`no-var`: report on the `var` keyword, not the entire var declaration

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_lint#1200]
### Lint Name

`no-var`

### Code Snippet

```ts
var foo = "1";
```

### Expected Result

The lint is reported here:

```ts
var foo = "1";
^^^
```

### Actual Result

The lint is reported here:

```ts
var foo = "1";
^^^^^^^^^^^^^^
```

### Version

```
deno 1.37.1 (release, aarch64-apple-darwin)
v8 11.8.172.3
typescript 5.2.2
```
